Makefile

```diff
@@ -69,38 +71,60 @@ test: unittest pep8
 
 ## Packaging
 
-package: cleanbuild
-	# PYDUNDASVER := $(shell $(PY3) -c 'from pydundas import __version__ as v; print(v)')
+package: cleanbuild getver
+	@echo Building Pydundas version $(PYDUNDASVER).
 	$(PY3) setup.py sdist bdist_wheel
+	@echo Built Pydundas version $(PYDUNDASVER).
+
+getver:
+PYDUNDASVER := $(shell $(PY3) -c 'from pydundas import __version__ as v; print(v)')
 
 ## Upload
 
+pypiversion: getver
+	@echo Module version: $(PYDUNDASVER).
+	@echo Latest version on test.pypi: $(shell curl -s 'https://test.pypi.org/pypi/pydundas/json' | jq -r '.info.version').
+	@echo Latest version on pypi: $(shell curl -s 'https://pypi.org/pypi/pydundas/json' | jq -r '.info.version').
+
+
 testpypi:
 	# Note: testpypi is defined in my $HOME/.pypitrc. Otherwise, replace --repository testpypi with
 	# --repository-url https://test.pypi.org/legacy/
+	@echo Latest version of Pydundas on test.pypi before upload: $(shell curl -s 'https://test.pypi.org/pypi/pydundas/json' | jq -r '.info.version').
 	$(PY3) -m twine upload --repository testpypi dist/*
+	@echo Latest version of Pydundas on test.pypi after upload: $(shell curl -s 'https://test.pypi.org/pypi/pydundas/json' | jq -r '.info.version').
+	@echo Note that it can take a few minutes to get the new version available. Check with make pypiversions.
 
 # Yes, test test: test the install from the test pypi repo.
-testtestinstall:
+testtestinstall: getver
 	# Install pydundas from tespypi
```

Comment:
Working so much with tez that you can't spell 'test' properly anymore? :stuck_out_tongue_winking_eye: